particles/ListView/ListView.ptcl

```diff
@@ -1,18 +0,0 @@
-/**
- * @license
- * Copyright (c) 2017 Google Inc. All rights reserved.
- * This code may only be used under the BSD style license found at
- * http://polymer.github.io/LICENSE.txt
- * Code distributed by Google as part of this project is also
- * subject to an additional IP rights grant found at
- * http://polymer.github.io/PATENTS.txt
- */
-
-ListView(in [Product] list)
```

Comment:
yep (at some point Scott renamed it to ShowProducts particle, but I guess he didn't delete this one, because it was used in the demo-flow-test)